src/Compilers/CSharp/Portable/Symbols/TypeParameterSymbol.cs

```diff
@@ -568,7 +568,7 @@ internal bool GetIsReferenceType(ConsList<TypeParameterSymbol> inProgress)
         }
 
         // https://github.com/dotnet/roslyn/issues/26198 Should this API be exposed through ITypeParameterSymbol?
-        internal bool? IsNotNullableIfReferenceType => GetIsNotNullableIfReferenceType();
+        internal abstract bool? IsNotNullableIfReferenceType { get; }
```

Comment:
Should this be `IsNonNullableIfReferenceType`? Generally we use Non in the APIs over Not. Want to make sure that's not significant here.

Reply:
> Should this be IsNonNullableIfReferenceType? Generally we use Non in the APIs over Not. Want to make sure that's not significant here.

This makes no difference to me. At the moment this is not a public API, we can revisit the name if we decide to make it public.

---
In reply to: [274057596](https://github.com/dotnet/roslyn/pull/34797#discussion_r274057596) [](ancestors = 274057596)